lib/swagger-explorer.ts

```diff
@@ -77,7 +77,7 @@ export class SwaggerExplorer {
   ) {
     let path = this.validateRoutePath(this.reflectControllerPath(metatype));
     if (modulePath) {
-      path = modulePath + path;
+      path = this.validateRoutePath(modulePath + path); // re-validate the route after adding the module path.
```

Comment:
perhaps, instead of leaving this comment (which is pretty straightforward), we should explain *why* it is needed

Reply:
Maybe adding additional information about that we need to re run that validation again since maybe the module path it self has url parameters segmentations, i think that would be enough.